app/src/main/res/values/strings.xml

```diff
@@ -8,4 +8,6 @@
     <string name="edit">Edit</string>
     <string name="edit_item">Edit Item</string>
     <string name="title">Title</string>
+    <string name="default_price">Price (default: 0.0)</string>
+    <string name="price">Price</string>
```

Comment:
Where is this string used?

Reply:
When creating a new item, in NewItemFragment I think

Reply:
"Price" is used when we are editing items and we completely clear the field where the price goes. In this case, the hint "Price" gets displayed in the editText box.